### PR TITLE
eigvalsh adhering to Tensorflow frontend

### PR DESCRIPTION
**Pull request by paulaehab**

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py [hunk 1]
```diff
@@ -45,3 +45,26 @@ def test_tensorflow_det(
         fn_tree="linalg.det",
         input=np.asarray(x, dtype=input_dtype),
     )
+@given(
```
Comment (Unevilicorn) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:48]: Linting issues in this test function, you can see the exact errors in the workflow

**Review by Unevilicorn — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py (hunk 1 above)
Comment (paulaehab) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:48]: How to pass all the tests? I don't find the reason of this error.
should I implement the same function for tourch and numpy ?

Comment (paulaehab) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:52]: Done finishing lint issues.
for the torch and numpy error it is not in this task or you want me to add it ?
and for the docstrings you put in the docs that I don't have to implement it, is it right ?

Comment (Unevilicorn) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:52]: Yep, just the lint issue, I'll get this merged now

**Pull request merged**